server/src/test/java/org/elasticsearch/indices/cluster/ClusterStateChanges.java

```diff
@@ -352,4 +352,9 @@ private ClusterState executeClusterStateUpdateTask(ClusterState state, Runnable
         assertThat(result[0], notNullValue());
         return result[0];
     }
+
+    @SuppressWarnings("unchecked")
+    private static <T, R, E extends Exception> CheckedFunction<T, R, E> anyCheckedFunction() {
```

Comment:
I feel like we've duplicated this enough times it might make sense to add it to `ESTestCase` or some other test utility.